frontend/src/components/molecules/MatchItem/MatchItem.tsx

```diff
@@ -184,3 +185,28 @@ export const MatchItem = (props: MatchItemProps,) => {
    )
 };
 
+const TeamHoverButtonComponent = (props: {
+   matchId: string | undefined,
```

Comment:
Moze byc druzyna bez 2 graczy? Albo bez teamId czy matchId? Typy musza nam pokazywac jak jest realnie :)

Reply:
Tak, mogą, jak mecz nie posiada żadnych drużyn - to i graczy brak!